pydoclint/utils/return_yield_raise.py

```diff
@@ -189,7 +192,7 @@ def _getRaisedExceptions(
 
 def _extractExceptionsFromExcept(
         node: ast.ExceptHandler,
-) -> Generator[str, None, None]:
+) -> Iterator[str]:
```

Comment:
`Iterator[str]` is sort of a shorthand for `Generator[str, None, None]`.  (And in Python 3.14, `Generator[str]` is finally recognized as a shorthand for `Generator[str, None, None]`.)

But to be consistent with existing functions in this file, I'll revert back to using `Generator[str, None, None]`.